### PR TITLE
Parser, service, serializer and viewset for drv register

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: location_register/converter/drv.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+from zeep.helpers import serialize_object
+
+from data_ocean.converter import Converter
+from location_register.models.drv_models import (DrvAto, DrvBuilding,
+                                                 DrvCouncil, DrvDistrict,
+                                                 DrvRegion, DrvStreet, ZipCode)
+
+
+class DrvConverter(Converter):
+    WSDL_URL = 'https://www.drv.gov.ua/ords/svc/personal/API/Opendata'
```
Comment (criticus) [inline location_register/converter/drv.py:14]: @Anntroy should this be in the all registers table in the db?

File: location_register/converter/drv.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    SETTINGS = Settings(strict=False, xml_huge_tree=True)
+    # dictionaries and lists for storing all objects from DB
+    all_regions_dict = {}
+    all_atos_dict = {}
+    all_zipcodes_dict = {}
+
+    # deleting 'м.', 'смт', 'с.', 'вул.' from string 
+    def clean_name(self, name):
```
Comment (criticus) [inline location_register/converter/drv.py:22]: consider moving this to utils, this is not the first time we use similar functionality

File: location_register/migrations/0010_auto_20200605_1305.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.9 on 2020-06-05 13:05
```
Comment (criticus) [inline location_register/migrations/0010_auto_20200605_1305.py:1]: consider squashing migrations into one

File: location_register/converter/drv.py (hunk 2 above)
Comment (IegorSoboliev) [reply location_register/converter/drv.py:22]: I plan to establish common utils files in the next PR

File: location_register/converter/drv.py (hunk 1 above)
Comment (IegorSoboliev) [reply location_register/converter/drv.py:14]: It depends on the purpose of having this table

File: location_register/migrations/0010_auto_20200605_1305.py (hunk 3 above)
Comment (IegorSoboliev) [reply location_register/migrations/0010_auto_20200605_1305.py:1]: done

File: location_register/converter/drv.py (hunk 1 above)
Comment (Anntroy) [reply location_register/converter/drv.py:14]: No. It's a new register, we need to add this information to the existing Register table.

File: location_register/models/drv_models.py [hunk 4]
```diff
@@ -4,11 +4,11 @@
 
 
 class DrvRegion(DataOceanModel):
```
Comment (Anntroy) [inline location_register/models/drv_models.py:6]: You have 8 migration files in the same PR!

File: location_register/converter/drv.py (hunk 1 above)
Comment (Anntroy) [reply location_register/converter/drv.py:14]: @IegorSoboliev I can't find it's register_id. I need this information to add to Register table.

Comment (IegorSoboliev) [reply location_register/converter/drv.py:14]: @Anntroy what is register_id?

**Pull request merged**